ivy/data_classes/container/base.py

```diff
@@ -1120,8 +1120,10 @@ def cont_from_disk_as_hdf5(
         """
         ivy.utils.assertions.check_exists(
             h5py,
-            message="You must install python package h5py in order to load hdf5 \
-            files from disk into a container.",
+            message=(
+                "You must install python package h5py in order to load hdf5            "
+                " files from disk into a container."
+            ),
```

Comment:
This is on of the artifacts of string preprocessing as well, I'm not sure of the behavior of `\ ` and I couldn't find anything online. Maybe it's a bug in our code as well

Reply:
I think the correct one should be 
```suggestion
            message=(
                "You must install python package h5py in order to load hdf5" \
                " files from disk into a container."
            ),
```